src/game_api/rpc.cpp

```diff
@@ -795,12 +801,13 @@ std::vector<uint32_t> entity_get_items_by(uint32_t uid, std::vector<ENT_TYPE> en
         const std::vector<ENT_TYPE> proper_types = get_proper_types(std::move(entity_types));
         if ((!proper_types.size() || !proper_types[0]) && !mask) // all items
         {
-            found.reserve(entity->items.size);
-            found.insert(found.end(), entity->items.uid_begin(), entity->items.uid_end());
+            found.reserve(found.size() + entity->items.size);

```

Comment:
there is not way that `found` will have any elements at this point is it? the whole `reserve` can be omitted for the same reason as in the first comment (not sure why i put it here)